src/gui/PhysicalGameStatePanel.java

```diff
@@ -295,114 +303,127 @@ public static void draw(Graphics2D g2d,
             }
         }
 
-        float eval0 = (evalFunction!=null ? evalFunction.evaluate(0, 1, gs):0);
-        float eval1 = (evalFunction!=null ? evalFunction.evaluate(1, 0, gs):0);
+        float eval0 = (evalFunction != null ? evalFunction.evaluate(0, 1, gs) : 0);
+        float eval1 = (evalFunction != null ? evalFunction.evaluate(1, 0, gs) : 0);
 
-        String info = "T: " + gs.getTime() + ", P0: " + unitCount0 + " (" + eval0 + "), P1: " + unitCount1 + " (" + eval1 + ")";
-        g2d.drawString(info, 10, dy-15);
+        String info = "T: " + gs.getTime() + ", P0: " + unitCount0 + " (" + eval0 + "), P1: " + unitCount1 + " ("
+                + eval1 + ")";
+        g2d.drawString(info, 10, dy - 15);
 
-//        g.drawString(gs.getTime() + "", 10, getHeight()-15);
+        // g.drawString(gs.getTime() + "", 10, getHeight()-15);
 
         AffineTransform t = g2d.getTransform();
 
-        if (panel!=null) {
-            panel.last_start_x = dx/2 - sizex/2;
-            panel.last_start_y = dy/2 - sizey/2;
+        if (panel != null) {
+            panel.last_start_x = dx / 2 - sizex / 2;
+            panel.last_start_y = dy / 2 - sizey / 2;
             panel.last_grid = grid;
             g2d.translate(panel.last_start_x, panel.last_start_y);
         } else {
-            int last_start_x = dx/2 - sizex/2;
-            int last_start_y = dy/2 - sizey/2;
+            int last_start_x = dx / 2 - sizex / 2;
+            int last_start_y = dy / 2 - sizey / 2;
             g2d.translate(last_start_x, last_start_y);
         }
 
         Color playerColor = null;
         Color wallColor = new Color(0, 0.33f, 0);
+        Color mineColor = new Color(0.33f, 0, 0);
```

Comment:
Is the `mine` color the same as the lava color?

Reply:
yes, I very recently added "lava"/converted things to make a Gridworld type of minigame. So I am using the same color for the mine and lava.